### PR TITLE
fix: Enable config to be installed in root of monorepo

### PR DESCRIPTION
**Pull request by tommoor**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = {
         alphabetize: {
           order: "asc"
         },
+        pathGroupsExcludedImportTypes: ["builtin"],
```
Comment (amccloud) [inline index.js]: I've read the docs on this but i'm still not entirely sure what omitting `external` will actually result in. 🤔

Comment (tommoor) [reply index.js]: I'm not totally clear, but without this line the imports are ordered incorrectly.

Comment (amccloud) [reply index.js]: Do we lose the ordering for external/npm dependencies?

Comment (tommoor) [reply index.js]: Double checked – ordering of external dependencies continues to work correctly!

**Comment by tommoor** [comment]
Merging thanks to ping from @rsylvian – `ui` has been using the prerelease for a while

**Pull request merged**